assets/stylesheets/units/_buttons.scss

```diff
@@ -0,0 +1,43 @@
+.button {
+  @include bold-font(20);
+
+  -webkit-appearance: none;
+  background-color: shade($green, 12.5%);
+  border: 0;
+  color: $white;
+  cursor: pointer;
+  display: inline-block;
+  letter-spacing: .5px;
+  margin: 0 ($baseline-grid-unit * 6) ($baseline-grid-unit * 3) 0;
+  padding: ($baseline-grid-unit * 3) ($baseline-grid-unit * 6);
+  text-align: center;
+  width: 100%;
+
+  @include media(tablet) {
+    width: auto;
+  }
+
+  &:link,
+  &:hover,
+  &:focus,
+  &:visited {
+    background-color: shade($green, 12.5%);
```

Comment:
Going back to an earlier point on shades not sure I like having these mixins scattered everywhere. If the current green doesn't fit then we should look at adjusting that or at least having set shades defined rather than using various different gradients. Will end up being quite scattered I feel.